website/content/v1.8/talos-guides/install/cloud-platforms/gcp.md

```diff
@@ -30,14 +30,14 @@ export REGION="us-central1"
 ### Create the Image
 
 First, download the Google Cloud image from a Talos [release](https://github.com/siderolabs/talos/releases).
-These images are called `gcp-$ARCH.tar.gz`.
+These images are called `gcp-$ARCH.raw.tar.gz`.
```

Comment:
@smira are these raw still or `.zst`?

Reply:
Still `.raw` (this is disk image suffix), e.g. https://github.com/siderolabs/talos/releases/tag/v1.8.0-alpha.0

GCP natively supports .tar.gz iirc, so we generate that by default.